test/e2e/velero_utils.go

```diff
@@ -195,21 +196,79 @@ func CheckRestorePhase(ctx context.Context, veleroCLI string, restoreName string
 func VeleroBackupNamespace(ctx context.Context, veleroCLI string, backupName string, namespace string) error {
 	backupCmd := exec.CommandContext(ctx, veleroCLI, "create", "backup", backupName, "--include-namespaces", namespace,
 		"--default-volumes-to-restic", "--wait")
+	backupCmd.Stdout = os.Stdout
```

Comment:
This is fine for now, but I think in the long run it would be good to have these commands take configurable `io.Writer`s or write the contents of `Stdout` and `Stderr` to buffers where the caller can decide where to redirect them as we do in the [`exec` util package](https://github.com/vmware-tanzu/velero/blob/main/pkg/util/exec/exec.go#L27-L54).

Reply:
Yes - actually, what I would like to do is fold this stuff into "velero debug" and call "velero debug" after each run and store things away in whatever format we use there.